packages/app/src/messages/components/thread-details-messages.component.tsx

```diff
@@ -58,6 +157,7 @@ const ThreadDetailsMessages: FunctionComponent<Properties> = ({
           return <MessageDayBubble key={id} {...messageDayBubble} />
         }}
       </ViewportList>
+      <div ref={newRef}></div>
```

Comment:
What means `newRef` 😉 New relative to another instance? What next if will occur the new one? 😉 
```suggestion
      <div ref={wrapperBottomRef} />
```

Reply:
Fixed on [aef71f2](https://github.com/mudita/mudita-center/pull/712/commits/aef71f29194ecd6b1ab1e0ef09ab5ffd06f447ac)